bitnami/kafka/templates/statefulset.yaml

```
@@ -190,7 +190,7 @@ spec:
             {{- end }}
             - name: KAFKA_CFG_LISTENERS
               {{- if .Values.listeners }}
-              value: {{ .Values.listeners }}
+              value: {{ include "kafka.listeners" . }}
```

I'd directly used "join" here and avoid adding new macros to `_helpers.tpl`:
```suggestion
              value: {{ join "," .Values.listeners }}
```